CHANGELOG.md

```diff
@@ -4,6 +4,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 # Upcoming
 
 ## StreamChat
+### 🐞 Fixed
+- Fix triggering user query calls whenever a new user was added in `UserListController` #[3184](https://github.com/GetStream/stream-chat-swift/pull/3184)
```

Comment:
Thinking if we should mention that the `UserListController` is no longer reactive, wdyt?

Reply:
Yeah, I was thinking about that too, but I was afraid it might warn customers for no important reason, since most of them won't care about this, but maybe better to be transparent, so I'll add this 👍

Reply:
Done ✅